### PR TITLE
Updated ghdocs output for DatePicker and ProgressIndicator.

### PR DESCRIPTION
**Pull request by wdo3650**

### CODE REVIEW

File: ghdocs/components/DatePicker.md [hunk 1]
```diff
@@ -98,7 +97,7 @@ State | Applied to | Result
 4. Verify that the component is working the same as in the sample above.
 
 ## Dependencies
-This component has no dependencies on other components. It does, however, require [pickadate.js 3.5.3](https://github.com/amsul/pickadate.js/releases/tag/3.5.3). Once you have downloaded the ZIP, you'll find `picker.js` and `picker.date.js` in the `lib` folder.
```
Comment (mikewheaton) [inline ghdocs/components/DatePicker.md]: This text about downloading the zip was just added recently. Could we get it back?

Comment (wdo3650) [inline ghdocs/components/DatePicker.md]: Okay, it's added back now.

File: ghdocs/components/ProgressIndicator.md [hunk 2]
```diff
@@ -9,6 +27,10 @@ Displays the progress of an operation when the percentage complete can be determ
 ![ProgressIndicator example](https://raw.githubusercontent.com/OfficeDev/office-ui-fabric-js/master/ghdocs/component_images/ProgressIndicator-default.png)
 
 
+## States
+Active<br/>
```
Comment (Jahnp) [inline ghdocs/components/ProgressIndicator.md]: Is the `<br />` here necessary since this is Markdown?

Comment (wdo3650) [inline ghdocs/components/ProgressIndicator.md]: This is removed

**Comment by ericthompson** [comment]
Approved

**Comment by mikewheaton** [comment]
Approved 🌖

**Pull request merged**